lib/tests/2wp.js

```diff
@@ -905,6 +914,130 @@ const execute = (description, getRskHost) => {
+      await fundRskAccountThroughAPegin(rskTxHelper, btcTxHelper, senderRecipientInfo2.btcSenderAddressInfo);
+
+      const senderRecipientInfo3 = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      await fundRskAccountThroughAPegin(rskTxHelper, btcTxHelper, senderRecipientInfo3.btcSenderAddressInfo);
+
+      const initial2wpBalances = await get2wpBalances(rskTxHelper, btcTxHelper);
+
+      const initialSender1AddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo1.btcSenderAddressInfo.address);
+      const initialSender2AddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo2.btcSenderAddressInfo.address);
+      const initialSender3AddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo3.btcSenderAddressInfo.address);
+
+      const pegout1ValueInSatoshis = MINIMUM_PEGOUT_AMOUNT_IN_SATOSHIS;
+      const pegout2ValueInSatoshis = MINIMUM_PEGOUT_AMOUNT_IN_SATOSHIS + 100_000;
+      const pegout3ValueInSatoshis = MINIMUM_PEGOUT_AMOUNT_IN_SATOSHIS + 150_000;
+      const totalPegoutValueInSatoshis = pegout1ValueInSatoshis + pegout2ValueInSatoshis + pegout3ValueInSatoshis;
+
+      const rskMemPoolTxHashes = await getRskMempoolTransactionHashes(rskTxHelper);
+      const initialRskMempoolTxHashesSize = rskMemPoolTxHashes.length;
+
+      // Act
+
+      const shouldMine = false;
+      const pegoutTransaction1Promise = sendTxToBridge(rskTxHelper, new BN(satoshisToWeis(pegout1ValueInSatoshis)), senderRecipientInfo1.rskRecipientRskAddressInfo.address, shouldMine);
+      const pegoutTransaction2Promise = sendTxToBridge(rskTxHelper, new BN(satoshisToWeis(pegout2ValueInSatoshis)), senderRecipientInfo2.rskRecipientRskAddressInfo.address, shouldMine);
+      const pegoutTransaction3Promise = sendTxToBridge(rskTxHelper, new BN(satoshisToWeis(pegout3ValueInSatoshis)), senderRecipientInfo3.rskRecipientRskAddressInfo.address, shouldMine);
+      
+      // Waiting for our 3 txs to reach the mempool before trying to mine them. + 3 because we are sending 3 pegouts.
+      const atLeastExpectedCount = initialRskMempoolTxHashesSize + 3;
+      await waitForRskMempoolToGetAtLeastThisManyTxs(rskTxHelper, atLeastExpectedCount);
+
+      // Now our 3 pegout transaction requests will get mined together.
+      await rskTxHelper.mine();
+      
+      // After mining, we can get the transaction receipts.
+      const pegoutTransaction1 = await pegoutTransaction1Promise;
+      const pegoutTransaction2 = await pegoutTransaction2Promise;
+      const pegoutTransaction3 = await pegoutTransaction3Promise;
+
+      // Assert
+
+      let bridgeStateAfterPegoutCreation;
+      
+      // Callback to get the bridge state after the pegout is created
+      const pegoutCreatedCallback = async () => {
+        bridgeStateAfterPegoutCreation = await getBridgeState(rskTxHelper.getClient());
+      };
```

Comment:
I might be missing something, what's the value of this callback? It will simply get the Bridge state and do nothing with it